pymc/tests/test_moment.py

```diff
@@ -0,0 +1,38 @@
+import aesara
+import numpy as np
+import pytest
+
+from aesara import tensor as at
+
+import pymc as pm
+
+from pymc.distributions.distribution import get_moment
+from pymc.distributions.shape_utils import to_tuple
+
+
+@pytest.mark.parametrize("size", [None, (), (2,), (3, 2)], ids=str)
```

Comment:
IIRC the None to () conversion happens in the `Distribution.__new__` and doesn't need to be tested here.